README.md

```diff
@@ -1,17 +1,130 @@
 # Pryv data types
 
-Standard event types used in Pryv.
+Events are the primary units of content in Pryv.io data model. Depending on its type, an event can represent anything related to a particular time (picture, note, data measurement, etc).
 
+We provide you with a list of **standard event types** used in Pryv that you can customize to suit your needs as explained below.
 
 ## Effective versions
 
-The types effectively in use are published on [our API site](http://api.pryv.com).
+The event types effectively in use are published on [our API site](https://api.pryv.com/event-types/).
+
+## How to customize your data types
+
+You can either [add issues](https://github.com/pryv/data-types/issues) or [fork and propose pull requests](https://github.com/pryv/data-types/fork). Please always include example use cases with your issues/pull requests.
```

Comment:
You must fork this repository, add the data types that you want to validate and host it on a URL where it will be loaded by Pryv.io on boot.